zerver/openapi/zulip.yaml

```diff
@@ -383,10 +383,13 @@ paths:
                               `{server_url}/user_uploads/{realm_id}/ab/cdef/temp_file.py`.
                           size:
                             type: integer
+                            nullable: true
```

Comment:
This field is I think just incorrectly nullable; I verified that the production code never has this equal null.

Reply:
in `zerver.tests.test_attachments.AttachmentsTests.test_list_by_user`
I got this response:
```
{
 "result": "success",
 "msg": "",
 "attachments": [
  {
   "id": 3,
   "name": "test.txt",
   "path_id": "foo/bar/test.txt",
   "size": null,
   "create_time": 1592683937000.0,
   "messages": []
  }
 ],
 "upload_space_used": 0
}

```

Reply:
Just saw your commit. Will remove it.

Reply:
I'm adding a prep commit to fix this in the code so we don't need to add this line.